sdk/python/feast/base_feature_view.py

```diff
@@ -152,15 +152,11 @@ def ensure_valid(self):
 
     def with_name(self, name: str):
         """
-        Renames this feature view by returning a copy of this feature view with an alias
-        set for the feature view name. This rename operation is only used as part of query
-        operations and will not modify the underlying FeatureView.
+        Returns a renamed copy of this base feature view. This renamed copy will only be
```

Comment:
Nit, 'should be' so that this comment can be used as guidance for future contributors

Reply:
good catch, updated